cmd/daprd/main.go

```diff
@@ -218,39 +223,42 @@ func main() {
 			}),
 		),
 		runtime.WithPubSubs(
-			pubsub_loader.New("redis", func() pubs.PubSub {
-				return pubsub_redis.NewRedisStreams(logContrib)
-			}),
-			pubsub_loader.New("natsstreaming", func() pubs.PubSub {
-				return natsstreaming.NewNATSStreamingPubSub(logContrib)
+			pubsub_loader.New("alicloud.rocketmq", func() pubs.PubSub {
+				return rocketmq_pubsub.NewRocketMQ(logContrib)
 			}),
 			pubsub_loader.New("azure.eventhubs", func() pubs.PubSub {
 				return pubsub_eventhubs.NewAzureEventHubs(logContrib)
 			}),
 			pubsub_loader.New("azure.servicebus", func() pubs.PubSub {
 				return servicebus.NewAzureServiceBus(logContrib)
 			}),
-			pubsub_loader.New("rabbitmq", func() pubs.PubSub {
-				return rabbitmq.NewRabbitMQ(logContrib)
+			pubsub_loader.New("gcp.pubsub", func() pubs.PubSub {
+				return pubsub_gcp.NewGCPPubSub(logContrib)
 			}),
 			pubsub_loader.New("hazelcast", func() pubs.PubSub {
 				return pubsub_hazelcast.NewHazelcastPubSub(logContrib)
 			}),
-			pubsub_loader.New("gcp.pubsub", func() pubs.PubSub {
-				return pubsub_gcp.NewGCPPubSub(logContrib)
-			}),
 			pubsub_loader.New("kafka", func() pubs.PubSub {
 				return pubsub_kafka.NewKafka(logContrib)
 			}),
-			pubsub_loader.New("snssqs", func() pubs.PubSub {
-				return pubsub_snssqs.NewSnsSqs(logContrib)
-			}),
 			pubsub_loader.New("mqtt", func() pubs.PubSub {
 				return pubsub_mqtt.NewMQTTPubSub(logContrib)
 			}),
+			pubsub_loader.New("natsstreaming", func() pubs.PubSub {
+				return natsstreaming.NewNATSStreamingPubSub(logContrib)
+			}),
 			pubsub_loader.New("pulsar", func() pubs.PubSub {
 				return pubsub_pulsar.NewPulsar(logContrib)
 			}),
+			pubsub_loader.New("rabbitmq", func() pubs.PubSub {
+				return rabbitmq.NewRabbitMQ(logContrib)
+			}),
+			pubsub_loader.New("redis", func() pubs.PubSub {
+				return pubsub_redis.NewRedisStreams(logContrib)
+			}),
+			pubsub_loader.New("snssqs", func() pubs.PubSub {
+				return pubsub_snssqs.NewSnsSqs(logContrib)
+			}),
 		),
 		runtime.WithNameResolutions(
```

Comment:
Do we need this one?

https://github.com/dapr/components-contrib/pull/743

Reply:
Good call out, lets leave this out until we get some clarification on the use case.

https://github.com/dapr/docs/pull/1310#issuecomment-842815489

Reply:
Did we forget  https://github.com/dapr/components-contrib/pull/743 ?